src/components/elements/dropdown.js

```diff
@@ -17,6 +17,7 @@ const DropdownContainer = styled.ul`
     border-radius: 4px;
     width: 152px;
     height: 32px;
+    z-index: 10;
```

Comment:
may i know why?

Reply:
because if you don't add z-index to the dropdown, in language switcher, when you expand it it goes under the footer's arrow.